kobo/settings/base.py

```diff
@@ -274,36 +274,48 @@
     ),
     'USER_METADATA_FIELDS': (
         json.dumps([
-            {'name': 'organization', 'required': False},
-            {'name': 'organization_website', 'required': False},
-            {'name': 'sector', 'required': False},
-            {'name': 'gender', 'required': False},
-            {'name': 'bio', 'required': False},
-            {'name': 'city', 'required': False},
-            {'name': 'country', 'required': False},
-            {'name': 'twitter', 'required': False},
-            {'name': 'linkedin', 'required': False},
-            {'name': 'instagram', 'required': False},
+            {'name': 'full_name', 'required': False,},
+            {'name': 'organization', 'required': False,},
+            {'name': 'organization_website', 'required': False,},
+            {'name': 'sector', 'required': False,},
+            {'name': 'gender', 'required': False,},
+            {'name': 'bio', 'required': False,},
+            {'name': 'city', 'required': False,},
+            {'name': 'country', 'required': False,},
+            {'name': 'twitter', 'required': False,},
+            {'name': 'linkedin', 'required': False,},
+            {'name': 'instagram', 'required': False,},
         ]),
         # The available fields are hard-coded in the front end
         'Display (and optionally require) these metadata fields for users. '
-        "Possible fields are 'organization', 'organization_website', "
+        "Possible fields are 'full_name', 'organization', 'organization_website', "
         "'sector', 'gender', 'bio', 'city', 'country', 'twitter', 'linkedin', "
-        "and 'instagram'",
+        "and 'instagram'.\n\r"
+        "To add another language, use 'some-other-language' as an example.",
         # Use custom field for schema validation
         'metadata_fields_jsonschema'
     ),
     'PROJECT_METADATA_FIELDS': (
         json.dumps([
-            {'name': 'sector', 'required': False},
-            {'name': 'country', 'required': False},
-            # {'name': 'operational_purpose', 'required': False},
-            # {'name': 'collects_pii', 'required': False},
+            {'name': 'sector', 'required': False, 'label': {
+                'default': 'Sector',
```

Comment:
I see the "Make label field optional" [commit](https://github.com/kobotoolbox/kpi/pull/4497/commits/bff0901589c91ef8cb511a8ea248158e6dddea44), but `label` is still included in these project metadata defaults even if it was removed from the user metadata defaults. ~~We should be consistent.~~ Er, it's not just consistency: if we override `label` by default, then the translated defaults in Transifex won't get used.

Still, we do need a way to show the `some-other-language` example.

Looking more, we have a prior problem with `MFA_LOCALIZED_HELP_TEXT`: the default (hard-coded) value we provide is not marked as a translatable string, and even if it were, there's no way for it to be localized [in the API response](https://github.com/kobotoolbox/kpi/blob/bff0901589c91ef8cb511a8ea248158e6dddea44/kpi/views/environment.py#L63-L71) unless the superuser translates it. (cc @noliveleger)

Reply:
OK, I see you put an example of using another language into the help text. This seems good :+1: